Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.yml

```diff
@@ -379,7 +379,71 @@ script:
       description: The number of engines that positively detected the indicator as
         malicious.
       type: string
-  dockerimage: demisto/python3:3.9.8.24399
+  - arguments:
+    - description: The value to scan
+      name: value
+      required: true
+    - auto: PREDEFINED
+      default: true
+      defaultValue: active
+      description: You can choose between passive and active scanning. Passive scans
+        fetch data without reaching out directly to the indicator, including performing
+        WHOIS and DNS requests. Active scans are more noisy; we'll do a quick port
+        scan and reach out to the indicator with a web browser.
+      name: scan_type
+      predefined:
+      - active
+      - passiv
+    description: Scan an indicator (IP/URL/Domain)
+    name: pulsedive-scan
+    outputs:
+    - contextPath: Pulsedive.Scan.qid
+      description: QID of the scan.
+      type: Number
+    - contextPath: Pulsedive.Scan.value
+      description: The value which was scanned.
+      type: string
+    - contextPath: Pulsedive.Scan.success
+      description: The success message.
+      type: string
+  - arguments:
+    - description: QID recieved from scan command
+      name: qid
+      required: true
+    description: Retrieve the Result
+    name: pulsedive-scan-result
+    outputs:
```

Comment:
please add here DBOT score as well - https://xsoar.pan.dev/docs/integrations/context-standards-mandatory#dbot-score

Reply:
In addition it seems like you are missing the URL https://xsoar.pan.dev/docs/integrations/context-standards-mandatory#url

Reply:
Done